server.js

```diff
@@ -74,6 +74,13 @@ function buildMentionSentence(reviewers) {
   );
 }
 
+function submitterMessageGenerator(submitter) {
+  return util.format(
+    '%s: Please review the checklist for your PR: (checklist goes here)',
```

Comment:
We don't need to mention the checklist since we will be attaching it in message.js.